HelpSource/Classes/Score.schelp

```diff
@@ -215,12 +217,12 @@ g = [
 	[0.3, [\s_new, \helpscore, 1003, 0, 0, \freq, 220]],
 	[1, [\c_set, 0, 0]] // finish
 	];
-f = File("score-test","w");
+f = File(Platform.defaultTempDir ++ "score-test","w");
 f.write(g.asCompileString);
 f.close;
 )
 
-z = Score.newFromFile("score-test");
+z = Score.newFromFile(Platform.defaultTempDir ++ "score-test");
```

Comment:
use `+/+` here and in the other spots above (i count 7)

Reply:
Thanks, and sorry this PR needs so much review for so little code